### PR TITLE
Add php v8.1 support

### PR DESCRIPTION
**Pull request by andreea-anamaria**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.2
   - 7.4
+  - 8.1
```
Comment (mihaileu) [inline .travis.yml]: let's  create another PR were we are switching to github acttions. You can use this as example: https://github.com/mmoreram/GearmanBundle/blob/master/.github/workflows/test.yaml

File: .travis.yml [hunk 2]
```diff
@@ -14,6 +15,7 @@ install:
 script:
   - vendor/bin/phpunit src/Tests --coverage-clover=coverage.clover
   - vendor/bin/phpstan analyse
+  - vendor/bin/rector process src --dry-run
```
Comment (mihaileu) [inline .travis.yml]: I would let rector to be used by dev one time only, I wouldn't commit it here. Install it globally or do not commit rector stuff

File: composer.json [hunk 3]
```diff
@@ -9,14 +9,15 @@
...
+        "php": "^7.2 | ^8.1",
         "guzzlehttp/guzzle": "^6.0|^7.0",
         "liuggio/statsd-php-client": "^1.0",
         "domnikl/statsd": "^2.0|^3.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^6.5|^7.0|^8.0",
-        "phpstan/phpstan": "^0.12.25"
+        "phpstan/phpstan": "^1.8",
+        "rector/rector": "^0.13.8"
```
Comment (mihaileu) [inline composer.json]: do not commit this

File: .travis.yml (hunk 2 above)
Comment (andreea-anamaria) [reply .travis.yml]: Fixed.

File: composer.json (hunk 3 above)
Comment (andreea-anamaria) [reply composer.json]: Fixed

File: .travis.yml (hunk 1 above)
Comment (andreea-anamaria) [reply .travis.yml]: I will look into and create another PR. Thank you.

**Pull request closed without merging**